pow/src/main/java/tech/pegasys/artemis/pow/api/PowEvent.java

```diff
@@ -14,5 +14,6 @@
 package tech.pegasys.artemis.pow.api;
 
 public interface PowEvent<T> {
+
```

Comment:
In order to use the ServiceAdapter on the other services, should PoWEvent be renamed and put in a different package?  Maybe `tech.pegasys.artemis.util.events`?

Reply:
Initially the service adapter only accepted classes that extend PowEvent but that restriction was lifted.  Right now any class type can be configured for both inbound and outbound.  It might be a good idea to add an interface that all events across services implement in the future though, and add the restriction back into the service adapter.